### PR TITLE
Issue #SC-1654 Initial draft for Trigger scheduler on demand

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

File: service/app/controllers/scheduler/SchedulerController.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import play.mvc.Http;
+import play.mvc.Result;
+
+import java.util.concurrent.CompletionStage;
+
+public class SchedulerController extends BaseController {
+ public CompletionStage<Result> startScheduler(Http.Request httpRequest) {
+  return handleRequest(
+          ActorOperations.START_SCHEDULAR.getValue(),
+          httpRequest.body().asJson(),
```
Comment (manzarul) [inline service/app/controllers/scheduler/SchedulerController.java]: Add job name validation as well, if user is passing three job name , and among three one job name is incorrect then provide him proper error and no job need to be run.
job should run if all provided details are correct.

File: service/app/controllers/scheduler/SchedulerController.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package controllers.scheduler;
+
+import controllers.BaseController;
+import org.sunbird.common.models.util.ActorOperations;
+import play.mvc.Http;
+import play.mvc.Result;
+
+import java.util.concurrent.CompletionStage;
+
+public class SchedulerController extends BaseController {
```
Comment (manzarul) [inline service/app/controllers/scheduler/SchedulerController.java]: Add unit test for same

File: service/app/controllers/scheduler/SchedulerController.java (hunk 1 above)
Comment (Hari-stackroute) [reply service/app/controllers/scheduler/SchedulerController.java]: It contains only one field, this particular validation is done in actor.

File: service/app/controllers/scheduler/SchedulerController.java (hunk 2 above)
Comment (Hari-stackroute) [reply service/app/controllers/scheduler/SchedulerController.java]: Added.

**Pull request merged**